cmd/pj-rehearse/main.go

```diff
@@ -32,19 +287,54 @@ func main() {
 	o := gatherOptions()
 	err := validateOptions(o)
 	if err != nil {
-		log.Fatal(err)
+		logrus.WithError(err).Fatal("invalid options")
+	}
+
+	jobSpec, err := getJobSpec()
+	if err != nil {
+		logrus.WithError(err).Fatal("could not read JOB_SPEC")
+	}
+
+	prFields := logrus.Fields{"org": jobSpec.Refs.Org, "repo": jobSpec.Refs.Repo, "PR": getPrNumber(jobSpec)}
```

Comment:
There's a helper upstream that gives you fields for spec

Reply:
Thanks, I'll have a look!

Reply:
`pjutil.ProwJobFields`

Reply:
Unfortunately that takes `ProwJob` and not `ProwJobSpec` :( We **could** make the former from the latter and then use that, but IMO that's not worth it just to extract few values.

Reply:
Very useful to have standard fields in logging for machine-driven filtering etc

Reply:
Using identical field names here without refactor SGTM for now

Reply:
Addressed in https://github.com/openshift/ci-operator-prowgen/pull/49/commits/c8b379bfea9953128bf5db1957e6822b5091829e by using names for the fields imported from `test-infra`.